allennlp/commands/evaluate.py

```diff
@@ -62,6 +62,9 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
         evaluation_data_file.add_argument('--evaluation_data_file',
                                           type=str,
                                           help=argparse.SUPPRESS)
+        subparser.add_argument('--weights-file',
+                               type=str,
+                               help='overrides which weights file to use')
```

Comment:
Should this be a full path or a relative path in the serialization directory?  Currently it needs to be a full path.  It's fine with me either way, as long as it's documented here.

Reply:
I like it being a full path, because then it's easy to grab the right file with autocomplete.

Reply:
Yeah, that's seems reasonable, I'd just be sure to note that in this help string.